### PR TITLE
Fix `unit.test_spm` for Windows

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Fixes some issues with imports and windows specific user and group values. This only fixes the test... I don't think it fixes SPM on Windows

### What issues does this PR fix or reference?
https://github.com/saltstack/salt-jenkins/issues/439

### Tests written?
No

### CODE REVIEW

**Review by gtmanfred — APPROVED** [review]

File: salt/spm/__init__.py [hunk 1]
```diff
@@ -491,10 +494,20 @@ def _install_indv_pkg(self, pkg_name, pkg_file):
...
-        uname = pwd.getpwuid(uid)[0]
-        gname = grp.getgrgid(gid)[0]
+        import salt.utils
+        if salt.utils.is_windows():
+            import salt.utils.win_functions
+            cur_user = salt.utils.win_functions.get_current_user()
+            cur_user_sid = salt.utils.win_functions.get_sid_from_name(cur_user)
+            uid = self.opts.get('spm_uid', cur_user_sid)
+            gid = self.opts.get('spm_gid', cur_user_sid)
+            uname = cur_user
```
Comment (cachedout) [inline salt/spm/__init__.py]: This is a lot of unnecessary assignment here. Why aren't we just calling this variable `uname` up in L500?

File: salt/spm/__init__.py [hunk 2]
```diff
@@ -710,7 +723,7 @@ def _create_repo(self, args):
             raise SPMInvocationError('A path to a directory must be specified')
 
         if args[1] == '.':
-            repo_path = os.environ['PWD']
+            repo_path = os.getcwd()
```
Comment (cachedout) [inline salt/spm/__init__.py]: Will this handle Unicode properly? Would `getcwdu()` be better?

**Comment by twangboy** [comment]
@cachedout Added your suggestions... rebased...

**Comment by rallytime** [comment]
bump @techhat - can you review this when you get a moment?

**Comment by techhat** [comment]
Honestly, I am not the correct person to be reviewing Windows changes. The new commands used aren't familiar to me.

**Review by rallytime — APPROVED** [review]

**Pull request merged**